pom.xml

```diff
@@ -189,6 +189,8 @@
         <dep.airlift.version>268</dep.airlift.version>
         <dep.alluxio.version>2.9.6</dep.alluxio.version>
         <dep.antlr.version>4.13.2</dep.antlr.version>
+        <!-- temporary until 9.8 is released -->
+        <dep.asm.version>9.8-SNAPSHOT</dep.asm.version>
```

Comment:
this might break when the snapshot gets cleaned up .. but good for.. when we update we should remove repo again as well

Reply:
Yep. This will be merged in 6+ months so ;)